demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -1,22 +1,271 @@
+                         commands=['ad-get-user', 'ad-search'])
+    create_a_pack_entity(active_directory_query, 'script', 'ADGetUser')
+
+    # Create a pack called 'Pcysys' with 1 playbook.
+    pcysys = repo.create_pack('Pcysys')
+    create_a_pack_entity(pcysys, 'playbook', 'Pentera Run Scan')
+
+    # Create a pack called 'Indeni' with 1 playbook.
+    indeni = repo.create_pack('Indeni')
+    create_a_pack_entity(indeni, 'playbook', 'Indeni Demo')
+
+    # Create a pack called 'Pcysys' with 1 playbook.
+    slack = repo.create_pack('Slack')
+    create_a_pack_entity(slack, 'playbook', 'Failed Login Playbook - Slack v2')
+
+    # Create a pack called 'FeedAWS' with 1 integration.
+    feed_aws = repo.create_pack('FeedAWS')
+    create_a_pack_entity(feed_aws, 'integration', 'FeedAWS', commands=['aws-get-indicators'])
+
+    # Create a pack called 'FeedAutoFocus' with 1 integration.
+    feed_autofocus = repo.create_pack('FeedAutofocus')
+    create_a_pack_entity(feed_autofocus, 'integration', 'FeedAutofocus', commands=['autofocus-get-indicators'])
+
+    # Create a pack called 'ipinfo' with 1 integration.
+    ipinfo = repo.create_pack('ipinfo')
+    create_a_pack_entity(ipinfo, 'integration', 'ipinfo', commands=['ip'])
+
+    # Create a pack called 'DigitalGuardian' with 1 incident_field.
+    digital_guardian = repo.create_pack('DigitalGuardian')
+    create_a_pack_entity(digital_guardian, 'incident_field', 'digitalguardianusername', 'Digital Guardian Username')
+
+    # Create a pack called 'EmployeeOffboarding' with 1 incident_field.
+    employee_offboarding = repo.create_pack('EmployeeOffboarding')
+    create_a_pack_entity(employee_offboarding, 'incident_field', 'googledisplayname', 'Google Display Name')
+
+    # Create a pack called 'Phishing' with 3 incident_fields and 1 script.
+    phishing = repo.create_pack('Phishing')
+    create_a_pack_entity(phishing, 'incident_field', 'attachmentname', 'Attachment Name')
+    create_a_pack_entity(phishing, 'incident_field', 'emailfrom', 'Email From')
+    create_a_pack_entity(phishing, 'incident_field', 'emailsubject', 'Email Subject')
+    create_a_pack_entity(phishing, 'script', 'CheckEmailAuthenticity')
+
+    # Create a pack called 'CommonTypes' with 3 incident_fields 2 incident_types 5 indicator_fields 1 indicator_type.
+    common_types = repo.create_pack('CommonTypes')
+    ct_incident_field_ids = ['accountid', 'country', 'username']
+    ct_incident_field_names = ['Account Id', 'Country', 'Username']
+    ct_incident_type_ids = ['Network', 'Authentication']
+    ct_incident_type_names = ['Network', 'Authentication']
+    ct_indicator_field_ids = ['accounttype', 'adminname', 'tags', 'commontypes', 'adminemail']
+    ct_indicator_field_names = ['Account Type', 'Admin Name', 'Tags', 'Common Types', 'Admin Email']
+    for i in range(len(ct_incident_field_ids)):
+        create_a_pack_entity(common_types, 'incident_field', ct_incident_field_ids[i], ct_incident_field_names[i])
```

Comment:
for all similar loops
```suggestion
    for field_id, field_name in zip(ct_incident_field_ids, ct_incident_field_names):
        create_a_pack_entity(common_types, 'incident_field', field_id, field_name)
```